crates/re_data_ui/src/image.rs

```diff
@@ -226,13 +226,13 @@ pub fn tensor_summary_ui_grid_contents(
 
     re_ui
         .grid_left_hand_label(ui, "Data type")
-        .on_hover_text("Data type used for all individual elements within the tensor.");
+        .on_hover_text("Data type used for all individual elements within the tensor");
     ui.label(tensor.dtype().to_string());
     ui.end_row();
 
     re_ui
         .grid_left_hand_label(ui, "Shape")
-        .on_hover_text("Extent of every dimension.");
+        .on_hover_text("Extent of every dimension");
```

Comment:
we should add a lint for this in `scripts/lint.py`!

Reply:
I agree, but I got lazy because linting rust with regex is rather annoying. How do you feel about putting in place some (syn-based?) rust-based rust lint?